Engineering/DevOpsLoggingDetails.md

```diff
@@ -0,0 +1,116 @@
+# Logging and Monitoring
+
+This sections is designed to give a deeper understading of logging and monitoring practices.
+
+## Logging
+
+### Logging Levels
+
+Each language/logging library defines its own logging level. They all have an order of importance that usually goes from critical &rarr; error &rarr; warning &rarr; info &rarr; debug. As a user of the logging library we can define what is the minimum log level that will be processed. For instance, setting "warning" as minimum log level will cause logs in warning, error and critical levels to be processed, the rest will be discarded.
+
+Be concious about logging levels for production environment. You might want to enable debug logging during a troubleshooting session, otherwise use a higher level to avoid collecting too much data and slowing down the system.
+
+### Logging Categories
+
+A few logging libraries additional have the concept of categories. Categories allow us to define different logging levels for each subsystem of an application. It enables a scenario where we turn debug logging for the authentication subsystem to troubleshoot login failures while leaving all other subsystems on a higher log level to minimize data being collected.
```

Comment:
Would change to "A few logging libraries have the additional concept of categories"

Reply:
Wording has been made clearer following your suggestion